src/main/kotlin/app/batch/MessageProducer.kt

```diff
@@ -19,7 +21,7 @@ class MessageProducer {
         val dateStr = if (date != null) date as String else ""
         val type = jsonObject.get("@type") ?: "MONGO_UPDATE"
         return if (StringUtils.isNotBlank(dateStr)) {
-                """{
+            """{
                 |   "message": {
                 |       "@type": "$type",
                 |       "_id": $id,
```

Comment:
Does ID come out with the embedded structure here?

Reply:
"_id": {"exampleId": "aaaa1111-abcd-4567-1234-1234567890ab"} ..It looks like this Mark